lib/modules/wdpa/story_map_link_list_importer.rb

```diff
@@ -8,18 +8,8 @@ def import
       csv.shift # remove headers
 
       csv.each do |row|
-        wdpa_id = Integer(row[0]) rescue false
-        if wdpa_id
-          pa = ProtectedArea.find_by_wdpa_id(wdpa_id)
-        else
-          pa = ProtectedArea.find_by_slug(row[0])
-        end
-
-        unless pa.blank?
-          pa.story_map_link = row[1]
-          pa.save
-          puts "Added this link #{row[1]} to #{row[0]}"
-        end
+        StoryMapLink.where(wdpa_id: row[0], link: row[1])
```

Comment:
This will create a new record even if there's no ProtectedArea with that WDPA ID.
Can we please check if a PA exists before adding story links?
If a PA does not exist, it would be good to log it on the screen